### ISSUE TITLE
Disable wanted badge

### ISSUE THREAD

**Issue by christianvaes** [tomreinartz90/Sonarr-extention-chrome#5]
By request we would like to add an option to disable the wanted badge if possible.

**Comment by tomreinartz90** [comment]
This should be possible. I think this takes 2 hours to complete

**Comment by billbrasky82** [comment]
I asked about this on reddit but I would request making the badge as not showing up when there is 0 items wanted but show up when there is >0 items. Too expand on this I use the badges mainly to alert me as to when something doesn't get grabbed properly and having the 0 badge makes me think there is something missing all the time.

**Comment by christianvaes** [comment]
Good idea, we should make an option to enable, disable and only show if not 0.
